src/components/interfaces/HMI_API.xml

```diff
@@ -3178,6 +3178,9 @@
     <param name="locationImage" type="Common.Image" mandatory="false">
       <description>     Image / icon of intended location (if applicable and supported)   </description>
     </param>
+    <param name="deliveryMode" type="Common.DeliveryMode" mandatory="false">
```

Comment:
@anosach-luxoft  Please squash commits with API changes if API changes don't affect SDL build and work. If it affects, please squash API and logic commits